src/Components/Publishing/VideoPlayer/MuteUnmute.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+import { IconVideoMute } from "../Icon/IconVideoMute"
+import { IconVideoUnmute } from "../Icon/IconVideoUnmute"
+
+interface Props extends React.HTMLProps<HTMLSpanElement> {
+  isMuted: boolean
+  toggleMute: () => void
+}
+
+export class MuteUnmute extends Component<Props, null> {
+  shouldComponentUpdate(nextProps) {
+    return this.props.isMuted !== nextProps.isMuted
```

Comment:
Since the props being passed in are simple and can be compared shallowly, can use `React.PureComponent` instead of checking against `shouldComponentUpdate`.

Reply:
Oooo nice!